voice/voice-ai/x/SpeechRecognition/SpeechRecognition.swift

```diff
@@ -662,6 +675,9 @@ class SpeechRecognition: NSObject, ObservableObject, SpeechRecognitionProtocol {
             }
 
             // Now make the query to fetch the fact.
+            let endTime = Int64(Date().timeIntervalSince1970 * 1000000)
```

Comment:
SST time should be 0 here since no SST is done. The time measured here is more like `[SST-END, APP-SEND]`